core/src/main/java/org/bitcoinj/core/PeerGroup.java

```diff
@@ -1482,9 +1481,9 @@ public synchronized void onBlocksDownloaded(Peer peer, Block block, @Nullable Fi
 
         @Override
         public synchronized void run() {
-            if (blocksInLastSecond > 1) {
```

Comment:
All blocks are requested in response to an inv even during chain download, so that wouldn't work.

But OK, if you feel strongly about this, let's set it to zero. I don't care much either way. The old code tried not to print download speeds once caught up with the chain head so the new code does too, that's the reason it's done this way.